lib/.eslintrc.yaml

```diff
@@ -264,58 +264,27 @@ globals:
 overrides:
   - files:
       - ./*/*.js
-      - ./_*.js
-      - ./child_process.js
-      - ./cluster.js
-      - ./console.js
-      - ./constants.js
-      - ./events.js
-      - ./fs.js
-      - ./internal/assert.js
-      - ./internal/child_process.js
+      - ./*.js
```

Comment:
just confirming, this is not recursive, right?

Reply:
It's not, there are still 247 missing trailing commas files inside subdirs of `lib/internal` (my understanding is that `*` matches any char but the path separator, and `**` would match any char including the path separator).